api/siderolink/provision.proto

```diff
@@ -23,6 +23,8 @@ message ProvisionRequest {
     optional string node_unique_token = 4;
     // Talos Version (as string) of the node.
     optional string talos_version = 5;
+    // Wireguard over GRPC tunnel request
+    optional bool wireguard_over_grpc = 6;
 }
 
 message ProvisionResponse {
```

Comment:
should have a flag that tunnel got provisioned

should have a token (might be an address from "4" ULA network) to use with WireguardOverGRPC service

Reply:
It now has "grpc_peer_addr_port" which acts as both virtual IP and a token for streamer to allow access.